edxval/pacts/README.md

```diff
@@ -4,7 +4,7 @@ This readme contains the description of different files in the current directory
 * middleware.py: This contains an authentication middleware that adds an authenticated user to the requests that are made while running provider verification.
 * views.py: This will have provider state setup view which is hit during provider verification to setup appropriate DB state per interaction
 * utils.py: Different DB utilities used by views.py
-* video-encode-manager-edx-val.json: The only pact file is a duplicate of pact file from https://github.com/edx/video-encode-manager/blob/a4f4c12db0c3919858df1e2fa58ee74071bc264a/encode_manager/apps/core/tests/pact_tests/pacts/video-encode-manager-edx-val.json. This is temporarily needed until pact broker can be setup. Until then, any update in the original pact should be ported in the json in VAL.
+* video-encode-manager-edx-val.json: The only pact file is a duplicate of pact file from https://github.com/openedx/video-encode-manager/blob/a4f4c12db0c3919858df1e2fa58ee74071bc264a/encode_manager/apps/core/tests/pact_tests/pacts/video-encode-manager-edx-val.json. This is temporarily needed until pact broker can be setup. Until then, any update in the original pact should be ported in the json in VAL.
```

Comment:
video-encode-manager is a private repo in edX Github org. The org name change should not be done here.

Reply:
@DawoudSheraz thanks! I pushed a commit.